### ISSUE TITLE
Hugging Face OPUS EN-PT Model

### ISSUE THREAD

**Issue by mike-goitia** [Helsinki-NLP/OPUS-MT-train#33]
Curious if there are plans to upload the EN-PT Portuguese model to HF library of pre-trained models?
It seems that `Helsinki-NLP/opus-mt-en-pt` does not exist.

Thanks!

**Comment by jorgtied** [comment]
Yes, the plan is to convert as many models as possible to HF. Apologies for the slow progress - it is a matter of finding time and doing some final adjustments that we need to take care of for the conversion process. Stay tuned!
